.github/workflows/source_gcp.yml

```diff
@@ -65,6 +65,9 @@ jobs:
       - name: gen
         if: github.event_name == 'pull_request'
         run: make gen
+      - name: Debug git diff
```

Comment:
Do we want to keep this?

Reply:
I just added it since we're not sure why it's failing